system/locales/zh-cn.json

```
@@ -0,0 +1,243 @@
+{
+    "button_createpost": "发表新文章",
+    "button_savechanges": "保存更改",
+    "button_create": "创建",
+    "button_upload": "上传",
+    "button_edit": "编辑",
+    "button_filterapply": "应用并搜索",
+    "button_filterreset": "重置",
+    "button_trashpost": "移动至回收站",
+    "button_untrashpost": "恢复",
+    "button_deletepost": "永久删除",
+    "button_publishpost": "发表文章",
+    "button_savepost": "保存文章",
+    "button_deleteposts": "立即删除",
+    "button_viewpost": "查看",
+    "button_back": "返回",
+    "button_deletetag": "删除标签",
+    "button_deleteuser": "刪除用户",
+    "button_login": "登录",
+    "button_logout": "登出",
+    "button_finishwizard": "完成设置",
+
+    "notice_title": "通知",
+    "notice_autodeleteposts": "回收站中的文章会在30天后被完全删除。",
+    "notice_appearances_updated": "外观设置更新成功！",
+    "notice_injected_updated": "自定义代码设置更新成功！",
+    "notice_nagivation_created": "导航添加成功！",
+    "notice_nagivation_updated": "导航更新成功！",
+    "notice_photo_uploaded": "照片上传成功！",
+    "notice_photo_deleted": "照片删除成功！",
+    "notice_post_created": "文章创建成功。您可以继续编辑此文章。",
+    "notice_post_updated": "文章更新成功！",
+    "notice_post_trashed": "文章已移至回收站！",
+    "notice_post_untrashed": "文章已成功恢复到文章列表中！",
+    "notice_post_clear": "回收站中的文章已被永久删除！",
+    "notice_post_deleted": "文章删除成功！",
+    "notice_settings_updated": "设置更新成功！",
+    "notice_tag_created": "标签创建成功！",
+    "notice_tag_updated": "标签更新成功！",
+    "notice_tag_deleted": "标签删除成功！",
+    "notice_user_created": "用户创建成功！",
+    "notice_user_updated": "用户更新成功！",
+    "notice_user_deletedwithposts": "用户删除成功，关联文章已转移到其他用户。",
+    "notice_user_deleted": "用户及关联文章删除成功！",
+    "notice_site_private": "该站点当前为私有。您必须登录才能查看内容。",
+    "notice_post_incorrect": "文章密码错误。",
+    "notice_login_incorrect": "电子邮件或密码不正确。",
+    "notice_loggedout": "您已成功登出。",
+
+    "alert_deleteposts": "您确定要删除回收站中的所有文章吗？此操作无法撤销！",
+    "alert_deletephoto": "您确定要删除此照片吗？此操作无法撤销！",
+    "alert_deletepost": "您确定要永久删除[%s]文章吗？",
+    "alert_deletetag": "您确定要删除[%s]标签吗？这个操作无法复原!",
+    "alert_deleteuser": "您真的要删除[%s]用户吗？这个操作无法复原！",

```

```suggestion
    "alert_deletepost": "您确定要永久删除“%s”文章吗？",
    "alert_deletetag": "您确定要删除“%s”标签吗？这个操作无法复原!",
    "alert_deleteuser": "您真的要删除“%s”用户吗？这个操作无法复原！",
```

改用中國的引號，~~看起來比較專業一點~~